src/reducers/entities/posts.js

```diff
@@ -359,6 +359,31 @@ function handleRemovePost(posts = {}, postsInChannel = {}, postsInThread = {}, a
     return {posts: nextPosts, postsInChannel: nextPostsForChannel, postsInThread: nextPostsForThread || postsInThread};
 }
 
+function clearChannelPosts(posts, postsInChannel, postsInThread, action) {
```

Comment:
`posts` and `postsInChannel` seems to have nothing to do here.  Should it still need to pass here?

Reply:
Yup, They are not needed. kinda went with the existing pattern. Have a look at the changes code and let me know if this is any better.

Reply:
👍 I'm good with that.